app-modules/service-management/src/Filament/Resources/ServiceRequestResource/Pages/ListServiceRequests.php

```diff
@@ -93,6 +93,11 @@ public function table(Table $table): Table
                     ->searchable(['service_request_number', 'title'])
                     ->sortable()
                     ->description(fn (ServiceRequest $record): string => $record->title),
+                TextColumn::make('status.name')
+                    ->label('Status')
+                    ->searchable()
```

Comment:
Let's not make this `searchable` in the future if Product wants we can add a filter. But no need to add this to the search query.

Reply:
Removed